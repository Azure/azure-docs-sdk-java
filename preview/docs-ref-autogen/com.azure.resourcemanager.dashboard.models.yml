### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.dashboard.models"
fullName: "com.azure.resourcemanager.dashboard.models"
name: "com.azure.resourcemanager.dashboard.models"
summary: "Package containing the data models for DashboardManagementClient."
classes:
- "com.azure.resourcemanager.dashboard.models.ActionType"
- "com.azure.resourcemanager.dashboard.models.ApiKey"
- "com.azure.resourcemanager.dashboard.models.AutoGeneratedDomainNameLabelScope"
- "com.azure.resourcemanager.dashboard.models.AzureMonitorWorkspaceIntegration"
- "com.azure.resourcemanager.dashboard.models.DeterministicOutboundIp"
- "com.azure.resourcemanager.dashboard.models.GrafanaIntegrations"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafanaListResponse"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafanaProperties"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafanaPropertiesUpdateParameters"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafanaUpdateParameters"
- "com.azure.resourcemanager.dashboard.models.ManagedServiceIdentity"
- "com.azure.resourcemanager.dashboard.models.ManagedServiceIdentityType"
- "com.azure.resourcemanager.dashboard.models.OperationDisplay"
- "com.azure.resourcemanager.dashboard.models.OperationListResult"
- "com.azure.resourcemanager.dashboard.models.Origin"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpoint"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnectionProvisioningState"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointServiceConnectionStatus"
- "com.azure.resourcemanager.dashboard.models.PrivateLinkResourceListResult"
- "com.azure.resourcemanager.dashboard.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.dashboard.models.ProvisioningState"
- "com.azure.resourcemanager.dashboard.models.PublicNetworkAccess"
- "com.azure.resourcemanager.dashboard.models.ResourceSku"
- "com.azure.resourcemanager.dashboard.models.UserAssignedIdentity"
- "com.azure.resourcemanager.dashboard.models.ZoneRedundancy"
interfaces:
- "com.azure.resourcemanager.dashboard.models.Grafanas"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.Definition"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.DefinitionStages"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.DefinitionStages.Blank"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.DefinitionStages.WithSku"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.DefinitionStages.WithTags"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.Update"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.UpdateStages"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.UpdateStages.WithIdentity"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.UpdateStages.WithProperties"
- "com.azure.resourcemanager.dashboard.models.ManagedGrafana.UpdateStages.WithTags"
- "com.azure.resourcemanager.dashboard.models.Operation"
- "com.azure.resourcemanager.dashboard.models.Operations"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.Definition"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.DefinitionStages"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.DefinitionStages.WithGroupIds"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.DefinitionStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.DefinitionStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.Update"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.UpdateStages"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.UpdateStages.WithGroupIds"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.UpdateStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnection.UpdateStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.dashboard.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.dashboard.models.PrivateLinkResource"
- "com.azure.resourcemanager.dashboard.models.PrivateLinkResources"
desc: "Package containing the data models for DashboardManagementClient. The Microsoft.Dashboard Rest API spec."
metadata: {}
package: "com.azure.resourcemanager.dashboard.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-dashboard:1.0.0-beta.2
