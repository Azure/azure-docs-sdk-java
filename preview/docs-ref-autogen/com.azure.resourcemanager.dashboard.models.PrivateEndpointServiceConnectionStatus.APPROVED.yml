### YamlMime:JavaMember
uid: "com.azure.resourcemanager.dashboard.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.dashboard.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.dashboard.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.dashboard.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
  desc: "Static value Approved for PrivateEndpointServiceConnectionStatus."
type: "field"
metadata: {}
package: "com.azure.resourcemanager.dashboard.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-dashboard:1.0.0-beta.2
